libcudacxx/include/cuda/std/__iterator/concepts.h

```diff
@@ -451,7 +451,6 @@ _LIBCUDACXX_CONCEPT_FRAGMENT(
   __contiguous_iterator_,
   requires(const _Ip& __i)(
     requires(random_access_iterator<_Ip>),
-    requires(derived_from<_ITER_CONCEPT<_Ip>, contiguous_iterator_tag>),
```

Comment:
can we scope this change to just c++17?

Reply:
That is C++17 only, because C++20 uses plain concepts